src/main/kotlin/uk/gov/justice/probation/courtcaseservice/service/CaseWorkflowService.kt

```diff
@@ -42,6 +42,6 @@ class CaseWorkflowService(val hearingRepository: HearingRepository, val courtRep
                     courtCode
                 )
             }
-        return hearingOutcomeRepositoryCustom.findByCourtCodeAndHearingOutcome(courtCode, hearingOutcomeSearchRequest).flatMap { HearingOutcomeResponse.of(it) }
+        return hearingOutcomeRepositoryCustom.findByCourtCodeAndHearingOutcome2(courtCode, hearingOutcomeSearchRequest).flatMap { HearingOutcomeResponse.of(it.first, it.second) }
```

Comment:
Is this renaming intentional?

Reply:
No, I left the code untidy and raised PR carelessly as soon as test passed. Apologies